.github/dependabot.yml

```
@@ -7,3 +7,9 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
+  
+  - package-ecosystem: "npm"
+    directory: "/eng/common/tsp-client/"
```

```suggestion
    directory: "/eng/common/"
```

Let's do it for everything under `/eng/common`, we should probably do that for everything in the repo also but that can be separate.